### PR TITLE
refactor: make the schemas for barriers and locationsfile use id

### PR DESCRIPTION
**Pull request by s-bessey**

### CODE REVIEW

File: scripts/upload-data.js [hunk 1]
```diff
@@ -256,48 +304,61 @@ const main = async () => {
...
+
+  // barriers schemas for hospitalization hotspots
+  if (id === "hospitalization_hotspots") {
+    files.push({
+      filePath: statebarriersfile,
       extension: "json",
-      field: "locations",
-      isArray: true,
+      field: "state_barriers",
+      isArray: false,
```
Comment (eldu) [inline scripts/upload-data.js:316]: I think it still is an array?

**Review by eldu — CHANGES_REQUESTED** [review]
I left a comment, i think i'm requesting some changes? But more so.a question

File: scripts/upload-data.js (hunk 1 above)
Comment (s-bessey) [reply scripts/upload-data.js:316]: No! Already updated, i just forgot to commit it 🤦

File: scripts/upload-data.js [hunk 2]
```diff
@@ -100,6 +100,7 @@ const main = async () => {
   if (!dateRegex.test(date)) {
     warnAndExit(`ERROR! Incorrect date format. Use yyyy-mm-dd format: ${date}`);
   }
+  // start out with a const baseFiles
```
Comment (eldu) [inline scripts/upload-data.js:103]: we can remove this comment

File: scripts/upload-data.js [hunk 3]
```diff
@@ -139,10 +140,10 @@ const main = async () => {
...
           type: "number",
         },
-        {
-          name: "name",
-          type: "string",
-        },
+        // {
+        //   name: "name",
+        //   type: "string",
+        // },
```
Comment (eldu) [inline scripts/upload-data.js:146]: This is temporary right? We'll want to enable this back in once we get the data?

**Review by eldu — CHANGES_REQUESTED** [review]
I just tried moving the base of this to `hosp-hotspot` but that didn't work out so well. Perhaps merge main into `hosp-hotspot` and then set the base of this pr to `hosp-hotspot` ?

I left a comment on the name field saying that we'd want to enable that back in once we got data. So perhaps having this go to the hotspot feature branch is the way to go while we wait

**Review by eldu — APPROVED** [review]
🎉

**Pull request merged**